src/deploy/table_event.sql

```diff
@@ -50,10 +50,19 @@ CREATE FUNCTION maevsi.trigger_event_search_vector() RETURNS TRIGGER AS $$
 DECLARE
   ts_config regconfig;
 BEGIN
+  IF NOT (
+    NEW.name IS DISTINCT FROM OLD.name OR
+    NEW.description IS DISTINCT FROM OLD.description OR
+    NEW.language IS DISTINCT FROM OLD.language
+  )
+  THEN
+    RETURN NEW;
+  END IF;
+
```

Comment:
@sthelemann I pushed a new commit that replaces this early return with a limitation of the updated trigger to the relevant columns. Do you agree both do the same?